actionview/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+*   annotated_source_code returns an empty array so TemplateErrors without a
+    template in the backtrace are surfaced properly by DebugExceptions.
+
+    *Guilherme Mansur*
```

Comment:
If you are giving credit to @kaspth in the commit, his name should go here as well. 😄

Reply:
good catch! fixed.